samples/manager/system-test/manager.test.js

```diff
@@ -0,0 +1,438 @@
+// Copyright 2017 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const iot = require('@google-cloud/iot');
+const path = require('path');
+const {PubSub} = require('@google-cloud/pubsub');
+const assert = require('assert');
+const tools = require('@google-cloud/nodejs-repo-tools');
+const uuid = require('uuid');
+
+const topicName = `nodejs-iot-test-topic-${uuid.v4()}`;
+const registryName = `nodejs-iot-test-registry-${uuid.v4()}`;
+const region = 'us-central1';
+const projectId =
+  process.env.GOOGLE_CLOUD_PROJECT || process.env.GCLOUD_PROJECT;
+
+const cmd = 'node manager.js';
+const cwd = path.join(__dirname, '..');
+const installDeps = 'npm install';
+const rsaPublicCert = process.env.NODEJS_IOT_RSA_PUBLIC_CERT;
+const rsaPrivateKey = process.env.NODEJS_IOT_RSA_PRIVATE_KEY;
+const ecPublicKey = process.env.NODEJS_IOT_EC_PUBLIC_KEY;
+
+const iotClient = new iot.v1.DeviceManagerClient();
+const pubSubClient = new PubSub({projectId});
+
+before(async () => {
+  tools.run(installDeps, `${cwd}/../mqtt_example`);
```

Comment:
For these `tools.run` or async running, usually we add a block like this:
https://github.com/googleapis/nodejs-kms/blob/master/samples/system-test/kms.test.js#L26

As a matter of fact - looking through those KMS samples is a good example of how to do these

Reply:
There are no examples that really match my case, I'll disable the tests until this refactor is stable.